.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Why not specify a version? "latest" is variable but our server is not.

Reply:
Fair point - done: https://github.com/reactome/release-data-exporter/pull/3/commits/9b9bbd3120e3cc1f88bcc5f6802018dc4f50980e

Reply:
And is 18.04 the version of our servers?